### PR TITLE
resize

### PR DESCRIPTION
**Pull request by YanxunLi**
implement resize.
need tests.
先看一下output是不是对的 有问题我再修改

### CODE REVIEW

File: WatermarkDetectron.py [hunk 1]
```diff
@@ -23,35 +29,50 @@ def __load_model(self, pretrained=None):
...
-        # TODO
-        
-        return ...
+        if url.endswith('.png'):
+            img_tensor = tf.image.decode_png(requests.get(url).content, channels=3, name="png_reader")
+            img_tensor = tf.image.resize(img_tensor, [200, 200])
+        elif url.endswith('.jpg') or url.endswith('.jpeg'):
+            img_tensor = tf.image.decode_jpeg(requests.get(url).content, channels=3, name="jpeg_reader")
+            img_tensor = tf.image.resize(img_tensor, [200, 200])
+        else:
```
Comment (XueweiYan) [inline WatermarkDetectron.py:46]: I'm thinking if we can also include a case for gif images. Is it possible for them to have watermarks at the bottom-right corner for gifs?

File: RetinaNet.py [hunk 2]
```diff
@@ -24,16 +24,20 @@ def __load_data(self, dataset, num_images=200):
...
     
     """
     Helper method for training
     """
-    def __resize_image(self, height=200, width=200):
-
-        # TODO
-
-        return ...
+    def __resize_image(self, size=200):
```
Comment (XueweiYan) [inline RetinaNet.py]: keep height and width as two separate params as we may need the freedom to resize to a rectangle. Also, it seems that I forget to include the param for the image... where does im_pth come from in the code?

File: WatermarkDetectron.py (hunk 1 above)
Comment (XueweiYan) [reply WatermarkDetectron.py:46]: A possible solution is to keep the first frame of the gif as the reference, then process that just as a static image. But need some researches on this...

**Review by XueweiYan — CHANGES_REQUESTED** [review]
Sorry too busy tonight... Can't catch functionality errors if any, but pointed out two logical improvements in the comments. You can ignore the gif problem for now. Fix the height/width problem, and let's merge what we have to the repo first.

File: RetinaNet.py (hunk 2 above)
Comment (YanxunLi) [reply RetinaNet.py]: im_pth will be the missing param. I'll add it.

**Review by XueweiYan — APPROVED** [review]

**Pull request merged**